distributed/tests/test_scheduler.py

```diff
@@ -62,6 +64,17 @@
 occupancy = defaultdict(lambda: 0)
 
 
+@pytest.mark.skipif(
+    "CYTHONIZED" not in os.environ, reason="CYTHONIZED environment variable not set"
+)
+def test_cythonized():
+    # This checks whether or not CI builds (where we set the CYTHONIZED environment variable) are
+    # cythonized as expected
+    import distributed.scheduler
+
+    assert strtobool(os.environ["CYTHONIZED"]) == COMPILED
```

Comment:
Sorry missed dropping the `import` in that PR

```suggestion
    # This checks whether or not CI builds (where we set the CYTHONIZED environment variable) are
    # cythonized as expected
    assert strtobool(os.environ["CYTHONIZED"]) == COMPILED
```

Reply:
No worries -- we should be good now 👍